Assets/GlitchEscape/Scripts/Player/Controls/PlayerJumpController_Alt.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerJumpController_Alt : MonoBehaviour, IPlayerControllerComponent
+{
+    private PlayerController controller;
+    private new Rigidbody rigidbody;
+    private Animator animator;
+    private Player player;
+    public void SetupControllerComponent(PlayerController controller)
+    {
+        this.controller = controller;
+        player = controller.player;
+        rigidbody = player.rigidbody;
+        animator = player.animator;
+        player.input.Controls.Jump.performed += OnJump;
+    }
+
```

Comment:
- [ ] need to be able to enable / disable wall jump ability / logic in inspector (ie. turn it on / off for playtesting)
- [ ] wall jump needs inspector parameters to control how the wall jump behaves (distance / forces + direction, etc)

wall jump also needs a *lot* of work to make it feel / play well